### DISCUSSION TITLE
Domain verification status suspended

### DISCUSSION THREAD

**Discussion by fhydzaa** [domcloud/domcloud#79] [Q&A] [answered]
![Screenshot 2024-06-21 181848](https://github.com/domcloud/domcloud/assets/101610607/da13a701-9cf5-49a9-912a-2391d2b160fe)

why can't I use the domain that I purchased on domcloud after a few days of being able to use it and the verification status is now suspended

**Comment by willnode** [comment, marked as answer]
Our registrar requires you to confirm your email when registering a new domain. (the email should be the same when signing up through DOM Cloud).

If you can't see the email please chat us personally (Chat button on DOM Cloud) so I can resend the email.

**Reply by fhydzaa** [reply]
Thank you, I've found the verification email in spam
